test/tools/WebListener/README.md

```diff
@@ -13,12 +13,14 @@ dotnet WebListener.dll ServerCert.pfx password 8083 8084 8085 8086
 
 The test site can then be accessed via `http://localhost:8083/`, `https://localhost:8084/`, `https://localhost:8085/`, or `https://localhost:8086/`.
 
-The `WebListener.dll` takes 4 arguments: 
+The `WebListener.dll` takes 6 arguments: 
 
 * The path to the Server Certificate
 * The Server Certificate Password
 * The TCP Port to bind on for HTTP
-* The TCP Port to bind on for HTTPS
+* The TCP Port to bind on for HTTPS using TLS 2.0
```

Comment:
oh whoops.. yes 1.2. fixed in all instances.

Reply:
Please clarify for me : is Tls 1.3 - Tls 2.0 today?

Reply:
Kind of... as I understand it a minor version iteration of 1.3 is 2.0. They *should* have versioned 1.3 as 2.0 as it was a significant change from 1.2, 1.1, and 1.0. In any case, neither 1.3 or 2.0 is supported by `HttpClientHandler` at this time. My hope is that when they decide on a new Enum for all of this, they at least pick field names that make sense instead of `None` being "whatever default the system uses" and `Default` being "one version that is not widely supported by APIs and another that is out of date".